app/services/balance_transactions/creator.rb

```diff
@@ -0,0 +1,28 @@
+class BalanceTransactions::Creator
+  Args = Struct.new(:date, :comment, :amount, :user_id)
+
+  def initialize(args)
+    @args = Args.new(args[:date], args[:comment], args[:amount], args[:user_id])
+  end
```

Comment:
@marderer Could you please clarify what you you want to say?

Reply:
@sinpai You can use 
``` ruby
  def initialize(date:, comment:, amount:, user_id:)
    @date = date
    @comment = comment
    @amount = amount
    @user_id = user_id
  end
```
instead of 
``` ruby
  Args = Struct.new(:date, :comment, :amount, :user_id)

  def initialize(args)
    @args = Args.new(args[:date], args[:comment], args[:amount], args[:user_id])
  end
```
And use `@date` instead `args[:date]`. It is more easier